# RNA splicing projects in the Knowles lab 

1. My first python package LeafletSC is available on [github](https://github.com/daklab/LeafletSC/tree/main). LeafletSC is a binomial mixture model designed for the analysis of alternative splicing events in single-cell RNA sequencing data. It relies on splice junction counts to infer meaningful cell populations driven by splicing differences.
2. We released a new R package called Isoviz that allows users to clearly visualize splice junctions across transcript isoforms expressed in their cell line of interest. Several visualizations are available as well as a function to design gRNA sequences to target junctions of interest with CRISPR/Cas13 using our lab's deep learning model [TIGER](https://www.nature.com/articles/s41587-023-01830-8). The Isoviz R package is available on [github](https://github.com/daklab/isoviz). Find the pre-print [here]().
3. IsovizPY coming soon! 

# Course projects @ Columbia PhD program 

Here are some class projects that I have worked on so far: 

### Predicting COVID-19 RNA-protein interactions 

[https://github.com/msthilaire5/ecbm-e4060-covid-interactions](https://github.com/msthilaire5/ecbm-e4060-covid-interactions)

### Protein sequence-based pathogen assembly from RNA-seq

[https://github.com/karini925/VORF](https://github.com/karini925/VORF)

# Lymphoma research 

From May 2019 to July 2021, I worked in [Dr. Robert Kridel's](http://kridel-lab.ca/) lab at [UHN](https://www.uhnresearch.ca/institutes/pm), Toronto. There, I contributed to several manuscripts focused primarily on functional genomics research in Lymphoma. For example, I created a [shiny based website](https://lymphoma.shinyapps.io/NHL-ASCT-PI/) to visualize a patient risk model that combines the PET-CT scan based Deauville score and other clinical risk factors.
